x-pack/plugins/security_solution/public/attack_discovery/tour/video_toast.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButton, EuiIcon, EuiToast, EuiPortal, EuiText, EuiSpacer } from '@elastic/eui';
+import { css } from '@emotion/react';
+import React, { useCallback, useEffect } from 'react';
+import * as i18n from './translations';
+
+const VIDEO_CONTENT_HEIGHT = 160;
+const VIDEO_CONTENT_WIDTH = 250;
```

Comment:
Since we can't go fullscreen, and you can't really tell what is going on with the video at this size (though you can hear James' beautiful voice), I wonder if we should just show an image capture of the video instead of actually embedding the video and avoid all the useEffect logic.

Alternatively, the whole page should be blank for the user, so could either make the portal bigger or even just use the empty state of the page? That's more discussion with design and there's not much time left so I don't know.

As a user though, I don't find any utility in watching the video in this teeny tiny portal, so would probably just watch it in a new tab.

Reply:
I asked design and they don't want to make the toast bigger. it would certainly be easier to do the `onClick` on a gif vs the hacked iframe solution I have. let me see if that is cool with design

@bojanasan @codearos what do you think about using a gif here instead of the autoplay video?

Reply:
gif or image... either way, would design be willing to provide a file?